lib/Api/LoginApi.php

```diff
@@ -84,17 +90,20 @@ class LoginApi {
 	 * @param Configuration   $config
 	 * @param HeaderSelector  $selector
 	 * @param int             $hostIndex (Optional) host index to select the list of hosts if defined in the OpenAPI spec
+	 * @param LoggerInterface|null $logger (Optional) logger instance for secure logging
 	 */
 	public function __construct(
 		?ClientInterface $client = null,
 		?Configuration $config = null,
 		?HeaderSelector $selector = null,
-		int $hostIndex = 0
+		int $hostIndex = 0,
+		?LoggerInterface $logger = null
```

Comment:
I know this is out of scope, but calling out the need to use the logger in the API classes. I'll make sure we have a story for that...

Reply:
https://bamboohr.atlassian.net/browse/SPN-2404